docs/source/cli_help/traverse.rst

```diff
@@ -65,6 +65,13 @@ It will pick up the walk from the current branch (unless ``--start-from=`` or ``
 Unlike with e.g. ``git rebase``, there is no special ``--continue`` flag, as ``traverse`` is stateless
 (doesn't keep a state of its own like ``git rebase`` does in ``.git/rebase-apply/``).
 
+``Traverse`` rebase/merge/push behaviour can also be customized for each branch separately using branch annotation qualifiers.
+There are ``rebase=no`` and ``push=no`` qualifiers that can be used to opt out of default behaviour (rebasing and pushing),
+there is ``merge=no`` qualifier that can be used to not use merge for certain branch when executing traverse with [-M|--merge] option
+and also a ``merge=yes`` qualifier that can be used to force ``traverse`` to use merge instead of rebase for certain branch (you don't need to provide [-M|--merge] flag).
+The qualifier can appear anywhere in the annotation but needs to be separated by a whitespace from any other character, e.g. ``some_annotation_text rebase=no push=no``.
+Qualifiers can only be overwritten by manually editing ``.git/machete`` file or modifying it with ``git machete e[dit]`` or by updating annotations with ``git machete anno``.
+
```

Comment:
I'd add sth like _Operations like `git machete github checkout-prs`_ (plus some other examples) _don't modify the qualifiers_